pymeasure/instruments/keithley/keithleyDAQ6510.py

```diff
@@ -33,15 +33,113 @@
 log.addHandler(logging.NullHandler())
 
 
+"""
+to implement
+channels as properties
+autogenerate channel list
+
+get current model of mux card
+
+possible implement full measurements via a "measure" class?
+so daq.measure.voltage
+"""
+
+
+
+# to implement:
+# :DISPlay:USER<n>:TEXT[:DATA] # displays user text on display.
+# :ABORt
+# :INITiate[:IMMediate]
+
+class ChannelState(Enum):
+    """
+    Enum for channel state to make it clearer in code if opening or closing channels.
+    """
+    OPEN = False
+    CLOSED = True
+
+
+class MuxChannel(Channel):
+    """mux relay channels"""
+
+    def __init__(self, chan_number):
+        self.chan_num = chan_number
```

Comment:
Channels already have an id as input parameter.

Please use `super.__init__` call in the init, in order to use the configuration of the Channel parent class.